ocs_ci/deployment/deployment.py

```diff
@@ -1798,20 +1743,42 @@ def deploy(self):
         config.switch_acm_ctx()
         # Create openshift-dr-system namespace
         run_cmd_multicluster(
-            f"oc create -f {constants.OPENSHIFT_DR_SYSTEM_NAMESPACE_YAML} "
+            f"oc create -f {constants.OPENSHIFT_DR_SYSTEM_NAMESPACE_YAML} ",
         )
+        self.deploy_dr_multicluster_orchestrator()
+        # Please create this only on ACM
```

Comment:
This comment is probably more useful being in the `OPENSHIFT_DR_SYSTEM_OPERATORGROUP` yaml file

Reply:
done, commented in yaml as well